strings/roundstart_hints.txt

```diff
@@ -140,3 +140,22 @@ Many items do more damage thrown than used in melee. Throw a chair at the traito
 You can only throw a standing person with an aggressive (upgraded) grab, but if someone is knocked down, you can throw them with a passive grab!
 Getting thrown into a vending machine can be quite painful! Watch where you fistfight.
 Sliding (block with an empty hand and throw at a direction) into objects on the ground causes you to kick them in that direction.
+Veteran players are known to choose staff assistant as their job for the shift. Don't underestimate people in grey jumpsuits!
+Engineers are responsible for power generation and station integrity, but they also have the MechComp system, which allows them to build wonderous and bizarre automated devices.
+Botanists don't just grow food and medicine; some plants are extremely dangerous, either as poisons or as creatures in themself!
+The bartender has a shotgun for keeping the peace when Security isn't on the scene. Some bartenders are known to replace the rubber slugs with deadlier loads.
+There are many improvised weapons you can make from the components in various tool storages. Check the construction page on the wiki.
```

Comment:
Hmm, maybe we could have some tips use HTML links that open in your browser.

Reply:
I pushed a commit that lets you include links as such:
```
<a href="byond://winset?command=.openlink https://google.com">foo</a>
```
Could be nice to link the construction page and the job pages I think.